yellowbrick/text/postag.py

```diff
@@ -325,20 +325,12 @@ def draw(self, **kwargs):
             self._pos_tags = np.array(self._pos_tags)[idx] 
             pos_tag_counts = pos_tag_counts[:,idx]
 
-        xidx = np.arange(len(self._pos_tags))
         if self.stack:
-            colors = resolve_colors(n_colors=len(self.labels_), 
-                                    colormap=self.colormap, 
-                                    colors=self.colors)    
-            #layer by layer plot for stacked bar chart
-            prev = np.zeros(len(self._pos_tags))
-            for index,element in enumerate(pos_tag_counts):
-                self.ax.bar(xidx, 
-                            element, 
-                            bottom = prev, 
-                            color = colors[index])
-                prev+=element
+            bar_stack(pos_tag_counts, ax=self.ax, labels=list(self.labels_), 
+                      ticks=self._pos_tags, colors=self.colors, 
+                      colormap = self.colormap)
```

Comment:
This certainly cleaned up a lot of the duplication! Nice one!

Reply:
It was a great idea to get a helper for stacked bar charts!